RAGchain/utils/linker/json_linker.py

```diff
@@ -42,12 +42,13 @@ def create_or_load_json(self):
             self.load_json()
 
     def put_json(self, id: Union[UUID, str], json_data: dict):
-        self.data[id] = json_data
+        self.data[str(id)] = json_data
         with open(self.json_path, "w") as f:
             json.dump(self.data, f)
 
     def get_json(self, ids: list[Union[UUID, str]]):
-        return [self.data.get(find_id) for find_id in ids]
+        str_ids = [str(find_id) for find_id in ids]
+        return [self.data.get(find_id) for find_id in str_ids]
```

Comment:
How about raise Error or Warning when there are no data for given id?
It is hard to track error when linker return None.